atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Value.java

```diff
@@ -120,7 +120,7 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         byte[] prefix = Arrays.copyOf(contents, Math.min(10, contents.length));
-        return "Value [contents=" + new String(BaseEncoding.base16().lowerCase().encode(prefix)) +
+        return "Value [contents=" + new String(PtBytes.encodeHexString(prefix)) +
```

Comment:
we're going to have to do another pass on toString methods to convert to `ToStringHelper`, I suspect. File an issue?